sencha/translate.py

```diff
@@ -340,32 +376,36 @@ def get_coding_score_line(self, description, jaccard, n_kmers, special_case, fra
 
     def score_reads_per_file(self, reads):
         """Assign a coding score to each read. Where the magic happens."""
-
-        scoring_lines = []
-
-        with screed.open(reads) as records:
-            for record in tqdm(records):
-                description = record["name"]
-                sequence = record["sequence"]
+        records = []
+        with screed.open(reads) as seqfile:
+            for read in seqfile:
+                description = read.name
                 if self.verbose:
                     logger.info(description)
+                records.append(tuple((description, read.sequence)))
```

Comment:
ah ok so it's not a named tuple, depending on memory usage this might actually be a better solution, otherwise we would have to store the extra space overhead of the named tuple. Would be interesting to look into what the storage differences actually are between named tuple and tuple, if it's fairly negligible it may still be good to have here.

Reply:
you brought up a great point - https://stackoverflow.com/questions/49419132/namedtuple-slow-compared-to-tuple-dictionary-class I am not sure how right the estimates in this link are but something to think about

Reply:
oh nice thanks for the reference! Best to keep as is then 👍